samba/docker-compose.yml

```
@@ -0,0 +1,31 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: samba_homepage_1
+      APP_PORT: 80
+
+  homepage:
+    image: joseluisq/static-web-server:2.32.0@sha256:c8345c2c469ba249cbc986b6155dc275ce86bcd30e0b29755fda4deb501bbddd
+    restart: on-failure
+    user: "1000:1000"
+    environment:
+      - SERVER_ROOT=/var/public
+    volumes:
+      - ${APP_DATA_DIR}/data/public:/var/public
+      - ${APP_DATA_DIR}/data/smb.conf:/var/public/smb.conf
+
+  server:
+    image: dockurr/samba:4.20.1@sha256:9a57887c0be1242ee2e1381653619a656820e1aee8e271885662d04c2c367be8
+    restart: on-failure
```

The user key is missing
```suggestion
    restart: on-failure
    user: "1000:1000"
```

---

It only worked for the Static Web Server, when I tried to do the same on the samba server it didn't started.
```
samba_server_1    | Failed to create user umbrel
samba_server_1    | adduser: permission denied (are you root?)
samba_server_1    | Failed to create user umbrel
```
https://github.com/getumbrel/umbrel-apps/pull/1161#discussion_r1646095174 the environment variable seems to be the only way (samba/docker-compose.yml:25-25).

---

https://github.com/dockur/samba/blob/master/samba.sh
I just read this file, the script need root access to add the user to the samba group and if the user don´t exist it will be created. It seems to be a good idea to have force group and force group, since the script will override it with the umbrel user and group.